library/X509/Job.php

```diff
@@ -99,7 +99,10 @@ private static function generateTargets(ConfigObject $jobDescription, array $hos
 //            }
             $ipv6 = filter_var($start_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) !== false;
             $subnet = $ipv6 ? 128 : 32;
-            $ip_count = 1 << ($subnet - $prefix);
+            $ip_count = pow(2, ($subnet - $prefix) - 2);
+
+            Logger::info("Found a total of '%d' IPs in CIDR '%s' to be scanned.", $ip_count, $cidr);
```

Comment:
Just use `Scanning '%d' IPs in the CIDR '%s'`. It's short and tells you exactly what the daemon will do.